core/src/main/java/bio/terra/pearl/core/dao/kit/KitRequestDao.java

```diff
@@ -20,4 +21,23 @@ public KitRequestDao(Jdbi jdbi) {
     public List<KitRequest> findByEnrollee(UUID enrolleeId) {
         return super.findAllByProperty("enrollee_id", enrolleeId);
     }
+
+    /**
+     * Find all kits that are not complete (or errored) for a study.
+     * This represents the set of in-flight kits that we want to keep an eye on in DSM.
```

Comment:
Looking for thoughts about naming. The DSM API refers to `JuniperKitRequest`, so "kit request" has made its way into naming in Juniper. However, the word "request" has a double meaning when building a web request to an endpoint to request a kit. Also, when the thing has been received and processed, is it still a "request"?

Therefore, I'm thinking of mostly dropping the word "request" from everything outside of the lowest DSM client layer. So, maybe use "sample kit" or just "kit".

In some contexts, I will say "sample collection kit" to be extra explicit about what this thing is from the perspective of a participant, but that feels a bit too verbose for code.